admin/worker/delete_unsued_assets.go

```diff
@@ -0,0 +1,80 @@
+package worker
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+const _defaultPageSize = 20
```

Comment:
1. Maybe 100? 
2. Also, rename to something like `_unusedAssetsPageSize` – it is not a default (since never overridden), and the scope covers multiple files that do pagination for different things (remember Go's scope is not per file, but per package)